assets/css/main.css

```
@@ -29,12 +29,13 @@ body {
 
 main {
     flex-grow: 1; /* Fill rest of empty body */
+    background-color: var( --white ) ;
 }
 
 /* Common content wrapper */
 .container {
     margin: 0 auto;
-    max-width: 1000px;
+    max-width: 1200px;
```

```suggestion
    max-width: 1230px; /* 1200px - width, 30px - paddings */
```

We've already had 15px side paddings here, so content would take the width just 1170px here (1200px - 2*15px = 1170px). We need to compensate the difference